.github/workflows/bazel.yml

```
@@ -81,20 +81,41 @@ jobs:
         # thereby only ever using the last successful cache version. This solution will result in a
         # few slower CI actions around the time cache is detected to be too large, but it should
         # incrementally improve thereafter.
-        - name: Ensure cache size
+        - name: Ensure cache size BAZEL_CACHE
+          # Only run on master to avoid slow build on PRs
+          if: github.event_name == 'push'
           env:
             BAZEL_CACHE_DIR: .${{ env.BAZEL_CACHE }}
-            BAZEL_CACHE_REPO_DIR: .${{ env.BAZEL_CACHE_REPO }}
+            # Use a 7GB threshold since actions/cache compresses the results, and Bazel caches seem
+            # to only increase by a few hundred megabytes across changes for unrelated branches.
+            # Uncompressed cache on master is looking to be around 6GB
+            BAZEL_CACHE_CUTOFF_MB: 7000
           run: |
             # See https://stackoverflow.com/a/27485157 for reference.
             EXPANDED_BAZEL_CACHE_PATH="${BAZEL_CACHE_DIR/#\~/$HOME}"
+
+            CACHE_SIZE_MB=$(du -smc $EXPANDED_BAZEL_CACHE_PATH | grep total | cut -f1)
```

```suggestion
            CACHE_SIZE_MB=$(du -sm $EXPANDED_BAZEL_CACHE_PATH | cut -f1)
```
If we give only one argument to `du`, we should be able to avoid `--total / -c` and with that the grep, afaiks.

---

_FYI a side note:_ `du` seems to be localized. I.e. on my German system `du -c` produces a line with the string `insgesamt` instead of `total`.